testnet/stacks-node/src/config.rs

```diff
@@ -1033,10 +1040,10 @@ impl NodeConfig {
             deny_nodes: vec![],
             local_peer_seed: local_peer_seed.to_vec(),
             miner: false,
-            mine_microblocks: false,
-            microblock_frequency: 5000,
+            mine_microblocks: true,
+            microblock_frequency: 30_000,
             max_microblocks: u16::MAX as u64,
-            wait_time_for_microblocks: 5000,
+            wait_time_for_microblocks: 120_000,
```

Comment:
I think this should be `60_000`, because the delay in burnchain detection may already cause a miner to lag too much when broadcasting their commitment.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2332/commits/c98187f9db9297f260f07fdd7c9a00f496c8e58a